app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -0,0 +1,64 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.model.Movie
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+
+data class MovieCardViewState(
+    val imageUrl: String?,
+    val id: Int,
+    val isFavorite: Boolean,
+    val title: String
+)
+
+@Composable
+fun MovieCard(modifier: Modifier,movieCardViewState: MovieCardViewState,onFavoriteButtonClicked: () -> Unit,
+              onClick: () -> Unit) {
```

Comment:
`onFavoriteButtonClicked` and `onClick` parameters are unused, please use them properly